meetings/2020-09-08.md

```diff
@@ -3,43 +3,111 @@
 ## [Teams Link](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2f_%23%2fl%2fmeetup-join%2f19%3ameeting_YWRkZjdiMGUtZWJlOC00OWMzLThlMTItZTk0Y2MyM2E1MWE0%40thread.v2%2f0%3fcontext%3d%257b%2522Tid%2522%253a%25220693b5ba-4b18-4d7b-9341-f32f400a5494%2522%252c%2522Oid%2522%253a%2522c27c6e98-e45a-45ff-aea5-7f10d6fe67c1%2522%257d%26anon%3dtrue&type=meetup-join&deeplinkId=e54b3969-3c7f-4efb-9cad-ee99cf639f86&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true)
 
 ### Attending
--
+
+- Jesse Mapel
+- Kris Becker
+- Stuart Robbins
+- Jay Laura
+- Ross Beyer
+- Robin Fergason
+- Audrie Fennema
+- Kristin Berry
+- Andrew Annex
+- Lauren Adoram-Kershner
+- Stuart Sides
+- Victor Silva
+- Adam Paquette
```

Comment:
I didn't check at the end of the meeting, so please make sure I didn't miss anyone.